scripts/core/platform.yml

```diff
@@ -24,6 +24,12 @@ details:
     - "Multiple calls to this function will return identical platforms handles, in the same order."
     - "The application may call this function from simultaneous threads, the implementation must be thread-safe"
 params:
+    - type: "$x_adapter_handle_t*"
+      name: "phAdapters"
+      desc: "[in][range(0, NumAdapters)] array of adapters to query for platforms."
+    - type: "uint32_t"
+      name: "NumAdapters"
+      desc: "[in] number of adapters pointed to by phAdapters"
```

Comment:
Feels odd to be passing in all the adapters here as a list, its a different approach to `urDeviceGet` only taking a single platform. I can see why this is a less intrusive way to do it though.

Reply:
Yeah, I did it this way to keep the changes as unobtrusive as possible, especially since `urPlatformGet` is in pretty much every example and test.

It also means fewer changes in the loader, which already had a special case for `urPlatformGet` where it looped over every adapter in the loader context. This PR just changes that to the adapters passed in.

If consistency with the other entry points is important I could look into changing this.

Reply:
I don't think its a blocker for this work but perhaps something to look into at a later point.